### PR TITLE
[Drools 4951] Update docs to explain security constraints

### PR DESCRIPTION
**Pull request by desmax74**
@jiripetrlik @danielezonca @mariofusco

### CODE REVIEW

File: docs/kjar/README.md [hunk 1]
```diff
@@ -23,18 +23,29 @@ Accordingly to your used module change
...
 [jdkhttp module](/jdkhttp/kubernetes/deployment.yaml)
+
+add the runAsAuser id and add the env vars
+change the value from the example with the correct runAsAUser ID, your KJARGAV and your MAVEN_MIRROR_URL with your nexus,  
+the MAVEN_LOCAL_REPO and MAVEN_SETTINGS_XML are the values with the default configuration in the docker container and UPDATABLEKJAR with value "true"
+to enable the update of the kjar
 ```yaml
-containers:
+      securityContext:
+        runAsUser: 1000170000
```
Comment (jiripetrlik) [inline docs/kjar/README.md]: Please change to "<id_user>" placeholder.

File: docs/kjar/README.md [hunk 2]
```diff
@@ -23,18 +23,29 @@ Accordingly to your used module change
...
+        runAsNonRoot: true
+      containers:
         - env:
           - name: UPDATABLEKJAR
             value: "true"
           - name: KJARGAV
-            value: "org.kie:sample-hacep-project:7.30.0.Final" 
+            value: org.kie.cloud:test-hacep-cloud-kjar:1.0.0
```
Comment (jiripetrlik) [inline docs/kjar/README.md]: I would suggest to use some placeholders for "KJARGAV" and "MAVEN_MIRROR_URL" instead of hardcoded values.

Comment (desmax74) [reply docs/kjar/README.md]: ok

File: docs/kjar/README.md (hunk 1 above)
Comment (desmax74) [reply docs/kjar/README.md]: ok

**Comment by desmax74** [comment]
jenkins test this

**Comment by desmax74** [comment]
jenkins test this

**Pull request merged**